client/src/Pos/Client/Txp/Util.hs

```diff
@@ -235,7 +235,7 @@ runTxCreator inputSelectionPolicy action = runExceptT $ do
 
 -- | Like 'makePubKeyTx', but allows usage of different signers
 makeMPubKeyTx
-    :: (HasConfiguration)
+    :: (HasProtocolMagic)
```

Comment:
Simpler constraints = easier to reuse <3

Reply:
Thank you! <3